osquery/tables/system/windows/certificates.cpp

```diff
@@ -307,6 +311,230 @@ void parseSystemStoreString(LPCWSTR sysStoreW,
   }
 }
 
+#pragma pack(push, 1)
+struct Header {
+  DWORD propid;
+  DWORD unknown;
+  DWORD size;
+};
+#pragma pack(pop)
+
+Status getEncodedCert(std::basic_istream<BYTE>& blob,
+                      std::vector<BYTE>& encodedCert) {
+  // See these links for details on this magic number:
+  // https://itsme.home.xs4all.nl/projects/xda/smartphone-certificates.html
+  // https://github.com/wine-mirror/wine/blob/f9301c2b66450a1cdd986e9052fcaa76535ba8b7/dlls/crypt32/crypt32_private.h#L146
+  static const DWORD CERT_CERT_PROP_ID = 0x20;
```

Comment:
Any specific reason though?
In this case probably it's fine either way, but in general since this is Windows specific code, it would be better imho to use its types, so that code doesn't break if anything changes.

Reply:
I can see both sides here. I feel like using the typedef is technically more idiomatic for Windows code, however, I can also see how we might want be consistent across the entire cross-platform codebase and just use standard types. There's also the issue of consistency within the existing Windows code, where much of it seems to use `unsigned long` currently.

I think for the short term (this PR), I'm fine with using `unsigned long`, and then later, if we decide to use typedefs, we can have a PR that just converts everything to that.